apps/voting/app/src/components/LocalIdentityBadge/LocalIdentityBadge.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Badge, IdentityBadge, font } from '@aragon/ui'
+import { IdentityContext } from '../IdentityManager/IdentityManager'
+
+function useIdentity(address) {
+  const [name, setName] = React.useState(null)
+  const { resolve, updates$, showLocalIdentityModal } = React.useContext(
+    IdentityContext
+  )
+
+  const handleNameChange = metadata => {
+    setName(metadata ? metadata.name : null)
+  }
+
+  const handleShowLocalIdentityModal = address => {
+    // Emit an event whenever the modal is closed (when the promise resolves)
+    return showLocalIdentityModal(address)
+      .then(() => updates$.next(address))
+      .catch(e => null)
```

Comment:
Do we need this `catch()` here? Does it make sense to propagate this reject to the caller?

Reply:
The linter will not allow us to push without it :(

Reply:
Aww shoot, we should disable this with the linter then :/

Reply:
Strangely the linter isn't giving me this error. But I do remember seeing it.